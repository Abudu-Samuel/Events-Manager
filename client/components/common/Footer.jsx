import React from 'react';

const Footer = () => (
  <div>
    <footer className="page-footer center-on-small-only default-color pt-0">
      <div className="footer-copyright">
        <div className="container-fluid">
                Â© 2017 Copyright: <a href="#"><strong> Events Manager.com
          </strong></a>
        </div>
      </div>
    </footer>
  </div>
);

export default Footer;
