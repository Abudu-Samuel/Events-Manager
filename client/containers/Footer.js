import React from 'react';

/**
 * creates Footer component
 * @returns {funct} Footer
 */
const Footer = () => (
    <div>
        <footer className="page-footer center-on-small-only default-color pt-0">
            <div className="footer-color">
                <div className="container">
                    <div className="row py-4 d-flex align-items-center">
                        <div className="col-md-6 col-lg-5 text-center text-md-left mb-4 mb-md-0">
                            <h6 className="mb-0 white-text">Get connected with us on social networks!</h6>
                        </div>
                        <div className="col-md-6 col-lg-7 text-center text-md-right">
                            <a className="icons-sm fb-ic ml-0"><i className="fa fa-facebook white-text mr-lg-4" /></a>
                            <a className="icons-sm tw-ic"><i className="fa fa-twitter white-text mr-lg-4" /></a>
                            <a className="icons-sm gplus-ic"><i className="fa fa-google-plus white-text mr-lg-4" /></a>
                            <a className="icons-sm li-ic"><i className="fa fa-linkedin white-text mr-lg-4" /></a>
                            <a className="icons-sm ins-ic"><i className="fa fa-instagram white-text mr-lg-4" /></a>
                        </div>
                    </div>
                </div>
            </div>
            <div className="container mt-5 mb-4 text-center text-md-left">
                <div className="row mt-3">
                    <div className="col-md-3 col-lg-4 col-xl-3 mb-r">
                        <h6 className="title font-bold"><strong>Events Manager</strong></h6>
                        <p className="text-justify">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco lab.
                        </p>
                    </div>
                    <div className="col-md-2 col-lg-2 col-xl-2 mx-auto mb-r">
                        <h6 className="title font-bold"><strong>Key Cities</strong></h6>
                        <p><a href="#!">Event centers in Lagos</a></p>
                        <p><a href="#!">Event centers in Ondo</a></p>
                        <p><a href="#!">Event centers in Abuja</a></p>
                        <p><a href="#!">Event centers in Ilorin</a></p>
                    </div>
                    <div className="col-md-3 col-lg-2 col-xl-2 mx-auto mb-r">
                        <h6 className="title font-bold"><strong>Useful links</strong></h6>
                        <p><a href="#!">Your Account</a></p>
                        <p><a href="#!">Available centers</a></p>
                        <p><a href="#!">Booking</a></p>
                        <p><a href="#!">Help</a></p>
                    </div>
                    <div className="col-md-4 col-lg-3 col-xl-3">
                        <h6 className="title font-bold"><strong>Contact</strong></h6>
                        <p><i className="fa fa-home mr-3" /> Anthony, LG 10012, NG</p>
                        <p><i className="fa fa-envelope mr-3" /> eventsmanager@gmail.com</p>
                        <p><i className="fa fa-phone mr-3" /> +234 6646 453</p>
                        <p><i className="fa fa-print mr-3" /> +234 6646 453</p>
                    </div>
                </div>
            </div>
            <div className="footer-copyright">
                <div className="container-fluid">
                Â© 2017 Copyright: <a href="#"><strong> Events Manager.com</strong></a>
                </div>
            </div>
        </footer>
    </div>
);

export default Footer;
