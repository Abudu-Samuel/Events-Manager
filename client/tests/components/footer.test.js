import React from 'react';
import { shallow } from 'enzyme';
import expect from 'expect';
import Footer from '../../components/common/Footer';

describe('<Footer />', () => {
  it('should render Footer correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
