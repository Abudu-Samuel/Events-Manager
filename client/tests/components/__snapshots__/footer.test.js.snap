// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render Footer correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <footer
        className="page-footer center-on-small-only default-color pt-0"
      >
        <div
          className="footer-copyright"
        >
          <div
            className="container-fluid"
          >
            © 2017 Copyright: 
            <a
              href="#"
            >
              <strong>
                 Events Manager.com
              </strong>
            </a>
          </div>
        </div>
      </footer>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="footer-copyright"
        >
          <div
            className="container-fluid"
          >
            © 2017 Copyright: 
            <a
              href="#"
            >
              <strong>
                 Events Manager.com
              </strong>
            </a>
          </div>
        </div>,
        "className": "page-footer center-on-small-only default-color pt-0",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="container-fluid"
          >
            © 2017 Copyright: 
            <a
              href="#"
            >
              <strong>
                 Events Manager.com
              </strong>
            </a>
          </div>,
          "className": "footer-copyright",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "© 2017 Copyright: ",
              <a
                href="#"
              >
                <strong>
                   Events Manager.com
                </strong>
              </a>,
            ],
            "className": "container-fluid",
          },
          "ref": null,
          "rendered": Array [
            "© 2017 Copyright: ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <strong>
                   Events Manager.com
                </strong>,
                "href": "#",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": " Events Manager.com",
                },
                "ref": null,
                "rendered": " Events Manager.com",
                "type": "strong",
              },
              "type": "a",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "footer",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <footer
          className="page-footer center-on-small-only default-color pt-0"
        >
          <div
            className="footer-copyright"
          >
            <div
              className="container-fluid"
            >
              © 2017 Copyright: 
              <a
                href="#"
              >
                <strong>
                   Events Manager.com
                </strong>
              </a>
            </div>
          </div>
        </footer>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="footer-copyright"
          >
            <div
              className="container-fluid"
            >
              © 2017 Copyright: 
              <a
                href="#"
              >
                <strong>
                   Events Manager.com
                </strong>
              </a>
            </div>
          </div>,
          "className": "page-footer center-on-small-only default-color pt-0",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="container-fluid"
            >
              © 2017 Copyright: 
              <a
                href="#"
              >
                <strong>
                   Events Manager.com
                </strong>
              </a>
            </div>,
            "className": "footer-copyright",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "© 2017 Copyright: ",
                <a
                  href="#"
                >
                  <strong>
                     Events Manager.com
                  </strong>
                </a>,
              ],
              "className": "container-fluid",
            },
            "ref": null,
            "rendered": Array [
              "© 2017 Copyright: ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <strong>
                     Events Manager.com
                  </strong>,
                  "href": "#",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": " Events Manager.com",
                  },
                  "ref": null,
                  "rendered": " Events Manager.com",
                  "type": "strong",
                },
                "type": "a",
              },
            ],
            "type": "div",
          },
          "type": "div",
        },
        "type": "footer",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
